### PR TITLE
Add new Win32 API emulation: wininet and kernel32

### PR DESCRIPTION
**Pull request by quangnh89**

### CODE REVIEW

**Comment by aquynh** [comment]
Nice! Please could you add one file in examples/, to test this new code?

Thanks.

**Comment by Dliv3** [comment]
Thanks for the PR. Could you implement the real behavior of the win32 apis in your PR instead of just return 1, e.g InternetReadFile？That will be very helpful.

**Comment by aquynh** [comment]
i fixed some conflicts.

@quangnh89 please add your name into the `Contributors` section of https://github.com/qilingframework/qiling/blob/master/CREDITS.TXT, thanks!

File: examples/adcache_x86_windows_debug.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+#
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+# Built on top of Unicorn emulator (www.unicorn-engine.org)
+#
+# LAU kaijern (xwings) <kj@qiling.io>
```
Comment (aquynh) [inline examples/adcache_x86_windows_debug.py]: please remove all the names in this file, and put your name in.

**Comment by quangnh89** [comment]
> Thanks for the PR. Could you implement the real behavior of the win32 apis in your PR instead of just return 1, e.g InternetReadFile？That will be very helpful.

Thank @Dliv3 . It's a great idea. I will implement this feature.

**Comment by Dliv3** [comment]
Thanks for the PR ! Merged! 👏

**Pull request merged**